src/css/profile/mobile/common/core.less

```diff
@@ -151,8 +151,9 @@
 		mask-border: url(images/0_Round_corner/round.svg) 77;
 		mask-border-width: 26 * @px_base;
 		border-radius: 26 * @px_base;
-		border: 1 * @px_base solid var(--content-area-line-color);
+		outline: 1 * @px_base auto var(--content-area-line-color);
```

Comment:
I don't know why we need outline? Can you show me use case?

Reply:
@TomaszLukawskiSam,
Yes. you are right. The outline is not needed. I will remove it.